components/org.wso2.carbon.identity.application.authenticator.fido/src/main/java/org/wso2/carbon/identity/application/authenticator/fido/FIDOAuthenticator.java

```diff
@@ -224,6 +577,11 @@ protected void initiateAuthenticationRequest(HttpServletRequest request,
         }
     }
 
+    private boolean isIDFInitiatedFromAuthenticator(AuthenticationContext context) {
+        return context.getProperty(FIDOAuthenticatorConstants.IS_IDF_INITIATED_FROM_AUTHENTICATOR) != null && (boolean)
+                context.getProperty(FIDOAuthenticatorConstants.IS_IDF_INITIATED_FROM_AUTHENTICATOR);
```

Comment:
Add a new line and Can't we use Boolean.parseBoolean(context.getProperty(FIDOAuthenticatorConstants.IS_IDF_INITIATED_FROM_AUTHENTICATOR)) here?

Reply:
Resolved via [59395d7](https://github.com/wso2-extensions/identity-local-auth-fido/pull/110/commits/59395d7a0c1dad27440c38d7ed768bb016868b68)